src/sentry/integrations/github/webhook.py

```diff
@@ -76,7 +76,6 @@ def __call__(self, event: Mapping[str, Any], host: str | None = None) -> None:
                     "external_id": str(external_id),
                 },
             )
-            logger.exception("Integration does not exist.")
```

Comment:
I think @armenzg added this for some specific reason, would be good to ask him.

Reply:
Let's turn it to a warning.

Ideally, we would investigate *why* we got into this state and prevent it from happening.
Or even tracking it as an analytic so we can see if something gets worse or fixed.

Reply:
@armenzg pretty sure this happens if you just uninstall the Github integration in Sentry. We already log the message at the info level above. If it's warning level, it's going to go to Sentry but info level just goes to logging.

Reply:
@scefali error, critical and exception goes to Sentry. warning does not.